### DISCUSSION TITLE
A way to make lines have rounded endings

### DISCUSSION THREAD

**Discussion by Terrariano27** [jgraph/drawio#4780] [Ideas]
It would be amazing to have this for some things, like making maps or things like that. Here is how it looks right now
![image](https://github.com/user-attachments/assets/a780d90a-0eba-45ba-9e92-a7cbb4eaa930)
And here is how would it look:
![image](https://github.com/user-attachments/assets/77030ad0-21af-4657-a87b-1867cdf3f307)

If there is a way to make this and i didn't know, please tell me !!

**Comment by davidjgraph** [comment]
<img width="851" alt="Capture d’écran 2024-12-04 à 20 58 42" src="https://github.com/user-attachments/assets/15207d2b-67e9-4404-90ea-2042d4327ab4">

**Reply by Terrariano27** [reply]
oh sorry, didn't know this :/
